src/App.js

```diff
@@ -22,6 +23,12 @@ class App extends Component {
 
   assignChosenMovie = (chosenMovie) => {
     this.setState({chosenMovie: chosenMovie})
+    this.setState({showingMain: false})
+  }
+
+  returnToMain = () => {
+    this.setState({showingMain: true})
+    this.setState({chosenMovie: null})
```

Comment:
Though I don't think we _need_ showingMain as a piece of state, I think it's worth mentioning that you can change multiple pieces of state with only one `setState` call. It might look something like:
`this.setState({showingMain: true, choseMovie: null})`